### DISCUSSION TITLE
Demangle and validate custom SSH Keys

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#835] [Feature Requests]
Copying the SSH keys into the input boxes can be challenging as often new lines and spaces are inserted during this process. It would be good if we could try to automatically fix all these problems.

We need some clear feedback mechanism to tell the user when the demangling did not work.

**Comment by leviwheatcroft** [comment]
Not sure if this is worth a separate issue but I simply couldn't get pasted keys to validate.

When the private / public key fields appear, the "Private Key Invalid" or "Public Key Invalid" invalid feedback is already shown, no matter what I did I couldn't get them to validate, so I couldn't "next". It's certainly possible that some weird characters were introduced when I copied the keys, but I was aware of this potential problem, so looking out for it, but still couldn't get this to work.

Eventually worked around it by generating a key.
